remoting/remoting-bolt/src/main/java/com/alipay/sofa/rpc/codec/bolt/SofaRpcSerialization.java

```diff
@@ -269,8 +269,9 @@ public <Request extends RequestCommand> boolean deserializeContent(Request reque
             long deserializeStartTime = System.nanoTime();
             try {
                 byte[] content = requestCommand.getContent();
-                if (content == null || content.length == 0) {
```

Comment:
Pls add soma test case to cover.

Reply:
Although I tried to add some unit tests, the SofaRpcSerialization class couples too much logic and it is difficult to construct a test scenario. I hope you understand it.

Reply:
updated.